description.md

```diff
@@ -34,6 +34,50 @@ Put differently, it shows how each content type is to be used in the various tea
 
 ### Practical Sessions
 
+Practical sessions are represented by hands-on activities that learners perform in order to acquire practical experience.
+These activities are typically supervized by a teacher that proposes the practical exercices and evaluates the learner's performance.
+The purpose of the evaluation is mainly to provide constructive feedback, as opposed to grading the student.
+
+Instances of practical sessions are represented by laboratories, seminars or workshops.
+The format of a practical session may include:
+
+- theoretical and practical explanations offered by the teacher.
+- practical activities performed by the learners.
+- debates and discussions between students and teacher or between students themselves regarding specific topics.
+- feedback from the teacher or peers.
+
+The presence and length of each of these activities during a practical session is influenced by the preference of the teacher.
+As such, the educational materials used as support for the practical sessions should offer the flexibility of being tailored depending on the teacher's needs.
+At the same time, they should be usable in the absence of a teacher as standalone learning materials for self-taught individuals that want to learn by themselves.
+
+To meet these constraints, educational materials for practical sessions should abide to the following format.
+A practical sessions is comprised of a series of concepts that the learner must assimilate.
```

Comment:
Both sentences serve as an introduction, I think it is clearer with only the second one.